setup/mail-postfix.sh

```diff
@@ -215,7 +215,7 @@ tools/editconf.py /etc/default/postgrey \
 cat > /etc/cron.monthly/mailinabox-postgrey-whitelist << EOF;
 #!/bin/bash
 # Mail-in-a-Box
-curl http://postgrey.schweikert.ch/pub/postgrey_whitelist_clients --output /etc/postgrey/whitelist_clients -ss
+curl https://postgrey.schweikert.ch/pub/postgrey_whitelist_clients --output /etc/postgrey/whitelist_clients -sS --fail > /dev/null 2>&1
```

Comment:
Should the output be to /etc/postfix/postgrey_whitelist_clients?

Reply:
On my box this is what the file is called.

Reply:
I don't have access to my box right now, I was just going by the comment at the top of: https://postgrey.schweikert.ch/pub/postgrey_whitelist_clients

The curl statement here is going to /etc/postgrey not /etc/postfix.

I can check later if needs be.